vital/kwiversys/CommandLineArguments.cxx

```diff
@@ -616,6 +616,8 @@ void CommandLineArguments::GenerateHelp()
       {
       str << std::endl;
       char argument[100];
```

Comment:
Do you think it is worth the effort to convert all this string handling to `std::stringstream`?

Reply:
I guess that depends on whether we're planning to use upstream KWSys fixes in kwiversys.  If we aren't then it might make sense to clean things up. Also, I guess since we named it kwiversys, updating from upstream will be somewhat difficult anyway ...

Reply:
That's a good point about pulling the latest version. Do you think it would be worth looking at the improvements made to the master version to see if we would benefit from them?